AsyncDisplayKit/Details/ASCollectionDataController.mm

```diff
@@ -143,15 +145,14 @@ - (void)prepareForInsertRowsAtIndexPaths:(NSArray<NSIndexPath *> *)indexPaths
 
 - (void)willInsertRowsAtIndexPaths:(NSArray<NSIndexPath *> *)indexPaths
 {
-  NSArray *keys = _pendingContexts.allKeys;
-  for (NSString *kind in keys) {
+  for (NSString *kind in _pendingContexts) {
     NSMutableArray<ASIndexedNodeContext *> *contexts = _pendingContexts[kind];
```

Comment:
I'm satisfied though, I'm going to migrate these over. I'm glad it turned out this way – Apple gave us a faster path. @appleguy